script/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "script"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+sqlx = { version = "0.7.2", features = ["runtime-tokio-rustls", "postgres", "macros"] }
+dotenv = "0.15.0"
+tokio = { version = "1", features = ["full"] }
+sqlx-macros = "0.7.2"
+chrono = "0.4.31"
+log = "0.4"
+env_logger = "0.10.1"
+
+[target.'cfg(target_os = "linux")'.dependencies]
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Hey @par4m, I still think it's something introduced in this repo, most probably the new Cargo.toml file with openssl. See for example https://github.com/reacherhq/check-if-email-exists/pull/1384, where the Github Action passes.

I'm reluctant to merge as-is, because then all the subsequent (now-passing) PRs will be red.

Could you try using a fixed version (like in cli I think)?